src/main/java/domain/RacingCarCurrentStatus.java

```diff
@@ -20,4 +20,8 @@ public Position position() {
     public boolean isAheadOf(Position maxPosition) {
         return this.position.isAheadOf(maxPosition);
     }
+
+    public boolean hasSamePositionWith(Position maxPosition) {
+        return this.position.equals(maxPosition);
+    }
```

Comment:
```suggestion
    public boolean isSameOf(Position position) {
        return this.position.equals(position);
    }
```

`maxPosition`은 호출하는곳에서의 이름아닐까요?
객체와 메소드에서 maxPosition이라는 개념을 추가해줄 필요가 무엇인가요?
사용하는 측면에서 maxPosition이라는 개념을 사용안한다면 해당 메소드를 사용못하는걸까요?

일반화된 접근을 해보면 어떨까요?

Reply:
놓치고 있었던 부분에 대한 피드백 감사합니다